homeassistant/components/met_lightning/__init__.py

```diff
@@ -0,0 +1,81 @@
+"""Support for Met Lightning data."""
+import logging
+from metno import LightningData
```

Comment:
Please add a blank line between standard library and 3rd party import groups. Group this with voluptuous.